fusion-cli/docs/fusionrc.md

```diff
@@ -21,6 +21,30 @@ module.exports = {
 
 **Please note that custom Babel config is an unstable API and may not be supported in future releases.**
 
+## `jsExtPattern`
+
+ By default this is `/\.jsx?$/`
+
+ For example, this enables to handle typescript files with addition to Babel plugins/preset:
```

Comment:
Grammer nitpick: 

For example, this enables Typescript support by adding the appropriate Babel preset: